content/how-to/cloud-servers/multifactor-authentication-from-the-myrackspace-portal/index.md

```diff
@@ -5,180 +5,171 @@ title: Multifactor authentication from the MyRackspace Portal
+
+8. In the Panel, enter your username and password. Then, enter the verification code from the OTP device that you paired with your account.
 
 ### Change the default MFA method
 
-If you configured your account with both SMS and OTP devices, you can
-select the default MFA method from the **My Profile & Settings** page.
+If you configured your account with both SMS and mobile passcode, you can select the default MFA method from the **My Profile & Settings** page.
 
 1. Log in to the [MyRackspace Portal](https://login.rackspace.com/).
 
-2. In the upper-right corner of the Cloud Control Panel, click the **user** menu
-    and select **My Profile & Settings**.
+2. In the upper-right corner of the MyRackspace Panel, click the **user** menu and select **My Profile & Settings**.
 
-3. In the **Security Settings** section, under **Multi-Factor Authentication**,
+<img width="947" alt="Dedicated 1" src="https://user-images.githubusercontent.com/98875667/161245007-03e8f041-943d-4062-8eba-80a4519a3955.png">
+
+3. In the **My Multi-factor devices** section, under **Multi-Factor Authentication**,
    select **Manage**.
 
-4. In the **Security Settings** section, select **Enable** for **Multi-factor
-    authentication**.
+<img width="553" alt="Dedicated 8" src="https://user-images.githubusercontent.com/98875667/161303017-65cef026-3d98-46d0-90e6-7806ac62b474.png">
+
+When a Mobile App is configured, the status in the **Manage** option is verified by default.
 
-5. Under **Method**, select **Switch to SMS** if you set up a Mobile App or **Switch
-   to Mobile App** if you set up SMS.
+<img width="405" alt="Dedicated 9" src="https://user-images.githubusercontent.com/98875667/161303331-ee292cea-1c28-401f-8316-21aa3c169148.png">
+
+
+Your default authentication method is currently set as SMS
+Click **Edit** option, then you can select either **Mobile Passcode Application** or **SMS**. 
+
+<img width="541" alt="Dedicated 11" src="https://user-images.githubusercontent.com/98875667/161304703-1c164800-189a-42d4-b0ef-55d3003c7aec.png">
 
 ### Configure account-wide MFA settings from an Administrator account
 
-Account administrators can update MyRackspace account-wide settings to require
-all users to authenticate by using MFA. When this setting is enabled, users can't
-access their accounts until they add and verify a device on their account.
+Account administrators can update MyRackspace account-wide settings to require all users to authenticate by using MFA. When this setting is enabled, users can't access their accounts until they add and verify a device on their account.
 
 To configure account-wide settings for MFA, use the following steps:
 
 1. Log in to the [MyRackspace Portal](https://login.rackspace.com).
 
 2. Click **Account**, and then select **User Management** from the menu.
 
+<img width="430" alt="Dedicated 12" src="https://user-images.githubusercontent.com/98875667/161504307-f61f1ac4-83a0-4ae4-9167-02479751b224.png">
```

Comment:
Move to folder

Reply:
Is there a real account number showing in this image? If its fake then fine, otherwise we should alter the number. I believe rohit just put `#1`. You can see what he did and do the same.

Reply:
Replace the image and account number was real so erased it and kept as #3.